CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Bugfix: Cinemachine3rdPersonFollow was not handling collision by default.
 - Bugfix: SaveDuringPlay saves only components that have the SaveDuringPlay attribute.
 - Regression fix: Entries in the custom blends editor in CM Brain inspector were not selectable.
+- Blends continue from interrupted blends progress.
```

Comment:
yes, this is lame.  How about:
"Bugfix: Reversing a blend in progress respects asymmetric blend times"
Not great either... but maybe a little more specific

Reply:
I fixed it